apps/admin/frontend/src/screens/tally_screen.tsx

```diff
@@ -133,13 +133,15 @@ export function TallyScreen(): JSX.Element | null {
         <H2>Cast Vote Record (CVR) Management</H2>
         {fileModeText && <P>{fileModeText}</P>}
         {isOfficialResults && (
-          <Button
-            variant="danger"
-            disabled={!hasAnyFiles}
-            onPress={() => beginConfirmRemoveFiles(ResultsFileType.All)}
-          >
-            Clear All Tallies and Results
-          </Button>
+          <P>
+            <Button
+              disabled={!hasAnyFiles}
+              onPress={() => beginConfirmRemoveFiles(ResultsFileType.All)}
+              variant="danger"
+            >
+              Clear All Tallies and Results
+            </Button>
+          </P>
```

Comment:
The spacing for this element is off without a `<P>`

Reply:
I noticed this on my branch yesterday, thanks for addressing!